scripts/17--html--forms/README.md

```diff
@@ -2,68 +2,93 @@
 
 ---
 
-- Concept
-    - Submitting data
-- Limitations of HTML & CSS:
-    - Need server-side code or JS
-- The Form tag
-    - action
-        - empty (same page)
-        - different URL
-    - method
-        - `POST`
-        - `GET`
-- Input & labels
-    - name: used server-side as key of the value
-    - `<label>`: accessibility
-    - id: for label
-
-    ```html
-    <label for="intext">First Name</label>
-    <input type="text" name="textin" id="intext" />
-    ```
-
-    - value: default values
-
-    ```html
-    <label for="city">Country</label>
-    <input type="text" name="city" id="city" value="United Kingdom" />
-    ```
-
-    - `placeholder`
-        - gives example input/more info
-        - better then pre-populating value (bad UX to have to delete it)
-
-- Text types
+#### Introduction
+
+- Forms are for submitting data. They allow the user to interact with your website.
+- This data is then processed and perhaps stored on the site's server
+- Therefore we can't do everything we want to with a form just in HTML and CSS
+- Even typing into Ecosia/Google search is using a form
+
+#### Form tag
+
+- `<form>`
+- formally and semantically defines your form, like a `<section>` or `<article>`
+- important attributes
+    - **action**
+        - defines the location (URL) where the form's collected data should be sent when it is submitted.
+        - if empty defaults to same page
+    - **method**
+        - defines which HTTP method to send the data with
+        - default is `GET`
```

Comment:
Worth demonstrating the difference between a `GET` and `POST`? Maybe not necessary, it's covered later in the course.